plugins/mcreator-localization/help/default/particle/max_age_diff.md

```diff
@@ -0,0 +1 @@
+This parameter controls how much actual maximal particle age can differ from the value above.
```

Comment:
can differ -> can vary in positive or negative value

Reply:
:heavy_check_mark: Fixed